ci/forge/Dockerfile.forge

```diff
@@ -0,0 +1,49 @@
+# syntax=docker/dockerfile:1.3-labs
+
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN <<EOF
+#!/bin/bash
+
+set -euo pipefail
+
+apt-get update
+apt-get upgrade -y
+apt-get install -y ca-certificates curl zip unzip sudo gnupg tzdata git
+
+# Add docker client APT repository
+mkdir -p /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+
+echo \
+  "deb [arch="$(dpkg --print-architecture)" signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+  "$(. /etc/os-release && echo "$VERSION_CODENAME")" stable" | \
+  sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+
+# Add NodeJS APT repository
+curl -fsSL https://deb.nodesource.com/setup_18.x | bash -
+
+# Install packages
+
+apt-get update
+apt-get install -y awscli docker-ce-cli nodejs build-essential
+
+# Needs to be synchronized to the host group id as we map /var/run/docker.sock
+# into the container.
+addgroup --gid 1001 docker
+
+# Install bazelisk
+npm install -g @bazel/bazelisk
+ln -s /usr/local/bin/bazel /usr/local/bin/bazelisk
+
+# A non-root user. Use 2000, which is the same as our buildkite agent VM uses.
+adduser --home /home/forge --uid 2000 forge --gid 100
+usermod -a -G docker forge
+
+EOF
+
+USER forge
```

Comment:
Discuss offline, but it seems that **_ci/ci.sh build_** will fail with this user

Reply:
what is the failure/error.

this is not intended for use to build ray yet, but only for things that are fully converted to use bazel.

building with root will also produce a lot of warnings.

Reply:
Something about permission defined when creating directory ./python/ray/generated; I'll send you the exact errors if I can recover it